dist/chrome/content/aboutPage.html

```diff
@@ -5,8 +5,9 @@
 <html>
 <head>
   <title>Lockbox</title>
+  <script src="chrome://lockbox/content/aboutPage.js" defer></script>
```

Comment:
Follow-up: yes, it is. I just need to run webpack to actually get this asset in the right place.